gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinServerAuthIntegrateTest.java

```diff
@@ -56,7 +56,7 @@ public Settings overrideSettings(final Settings settings) {
         final Settings.AuthenticationSettings authSettings = new Settings.AuthenticationSettings();
         authSettings.authenticator = SimpleAuthenticator.class.getName();
 
-        // use a credentials graph with one user in it: stephen/password
+        // use a credentials graph with two users in it: stephen/password and marko/rainbow-dash
```

Comment:
i suppose the addition of the `marko` user was the change here:

https://github.com/apache/tinkerpop/pull/1308/files#diff-a73da218a425ffa2b99f8c287c3ff3477426bb72748d0c21894d5067485b12a8R47

just a new user...is that right? perhaps now would be a smart time to add a `README.md` to `gremlin-server/data` to remind us what the contents of those files are. i'm not even sure what `gremlin-server/data/sample.kryo` contains anymore.

<img src="https://user-images.githubusercontent.com/384249/97305738-e8056b00-1833-11eb-8ae5-a6546fe177cf.png"  width="200" />

Reply:
Yes, I picked this user from the credentials-dsl example from the ref docs...
I better leave this to you, because this also relates to preparing the console and server distributions.

Reply:
Thanks, will do that.

Reply:
Done